### PR TITLE
Documentation edits (round 2)

### PR DESCRIPTION
**Pull request by hammondkd**
**Summary**

I have made many changes to clean up the documentation. Primarily these include mathematical expressions, but I also replaced parsed-literal with code-block LAMMPS in the Syntax sections. Any missing style descriptions for computes and fixes were added. I also included some changes that make the documentation more consistent and/or easier to parse for a non-human reader (e.g., replacing "[nocoeff]" with "keyword" and "zero or more keywords may be appended" and so forth).

I do not think I made any substantive changes to the documentation, only cleaning up math and wordsmithing edits plus those described above. However, the number of these edits is getting extreme, so I'm pushing these now so it is easier for the core developers to read them and take them in. I will continue working on this, continuing with fix styles, later.

These changes build cleanly with no warnings or errors using both "make html" and "make fasthtml" in the doc directory.

**Related Issue(s)**

Works toward #3371 (code clinic 2022, Task 4)

**Author(s)**

Karl Hammond, University of Missouri
hammondkd at missouri.edu

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

No change to the program itself are included, only to the documentation.

**Post Submission Checklist**

- [ ] The feature or features in this pull request is complete
- [ ] Licensing information is complete
- [ ] Corresponding author information is complete
- [ ] The source code follows the LAMMPS formatting guidelines
- [ ] Suitable new documentation files and/or updates to the existing docs are included
- [ ] The added/updated documentation is integrated and tested with the documentation build system
- [ ] The feature has been verified to work with the conventional build system
- [ ] The feature has been verified to work with the CMake based build system
- [ ] Suitable tests have been added to the unittest tree.
- [ ] A package specific README file has been included or updated
- [ ] One or more example input decks are included

### CODE REVIEW

**Comment by akohlmey** [comment]
@hammondkd thanks for this massive update. Ran this through the spell checker and fixed a few minor issues.
Found a misspelled Boltzmann (and removed the corresponding incorrect fixup in the false positives list).
With that I also changed some (more) Boltzmann constants from "k" to "k_B". I think with "k" often referring to force constants, using "k_B" is the better choice.

File: doc/src/compute_modify.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Syntax
          N = # of extra degrees of freedom to subtract
        *extra* syntax is identical to *extra/dof*, will be disabled at some point
        *dynamic/dof* value = *yes* or *no*
-         yes/no = do or do not re-compute the number of degrees of freedom (DOF) contributing to the temperature
+         yes/no = do or do not re-compute the number of degrees of freedom (dof) contributing to the temperature
```
Comment (akohlmey) [inline doc/src/compute_modify.rst]: Since "DOF" is an initialism, I prefer it to be in uppercase or at least "DoF".

Comment (hammondkd) [reply doc/src/compute_modify.rst]: The documentation was split on this; some used "dof" while others used "DOF"; I went with the first one I found, which was "dof," but I could change them all to DOF if that is preferable.

Comment (akohlmey) [reply doc/src/compute_modify.rst]: It is certainly *my* preference. I cannot speak for the other LAMMPS developers.

**Review by sjplimp — APPROVED** [review]

**Pull request merged**